teapots/choreographer-30fps/src/main/cpp/ChoreographerNativeActivity.cpp

```diff
@@ -337,11 +337,19 @@ void Engine::UnloadResources() { renderer_.Unload(); }
 /**
  * Initialize an EGL context for the current display.
  */
-int Engine::InitDisplay() {
+int Engine::InitDisplay(android_app *app) {
   if (!initialized_resources_) {
     gl_context_->Init(app_->window);
     LoadResources();
     initialized_resources_ = true;
+  } else if(app->window != gl_context_->GetANativeWindow()) {
+    assert(gl_context_->GetANativeWindow());
```

Comment:
Can we have some comments here?

e.g.
Re-initialize ANativeWindow. On some devices, ANativeWindow is re-created when the app is resumed.

Reply:
done, thanks!